virttest/qemu_vm.py

```diff
@@ -881,6 +881,10 @@ def set_dev_params(dev, dev_params,
                 dev.set_param("id", dev_id)
 
             dev_type = "virtio-rng-pci"
+            machine_type = self.params.get("machine_type")
```

Comment:
@Zhengtong  machine_type == None will cause 'TypeError'

Reply:
Oh ,yes.  Thanks for the tips. I will give it a default value if no machine_type got from configurations